PurchasesCoreSwift/SubscriberAttributes/SpecialSubscriberAttributes.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SpecialSubscriberAttributes.swift
+//  PurchasesCoreSwift
+//
+//  Created by César de la Vega on 6/17/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+// swiftlint:disable identifier_name
+@objc(RCSpecialSubscriberAttributes) public class SpecialSubscriberAttributes: NSObject {
+    @objc public static let email = "$email"
```

Comment:
I'm guessing that this is done this way so we can share it with obj-c, and we'll probably update it during the swift migration, right?

Reply:
yes exactly